std/math.d

```diff
@@ -2181,11 +2181,12 @@ creal expi(real y) @trusted pure nothrow @nogc
 T frexp(T)(T value, out int exp) @trusted pure nothrow @nogc
     if(isFloatingPoint!T)
 {
-    ushort* vu = cast(ushort*)&value;
+    Unqual!T vf = value;
```

Comment:
I meant the name `vf`.

Yes, using `in` or `const` makes sense here, because it reduces the number of template instantiations.